spec/requests/admin/email_styles_controller_spec.rb

```diff
@@ -14,6 +14,10 @@
     SiteSetting.remove_override!(:email_custom_css)
   end
 
+  it "is a subclass of AdminController" do
+    expect(Admin::EmailStylesController < Admin::AdminController).to eq(true)
+  end
```

Comment:
This assertion feels more like a unit test than a request spec. I think a test which checks that a non-admin user cannot access the endpoints here is more appropriately.